util/design/secded_gen.py

```diff
@@ -76,6 +78,9 @@
 # secded configurations
 SECDED_CFG_FILE = "util/design/data/secded_cfg.hjson"
 
+PROJ_ROOT = Path(__file__).parent.joinpath('../../')
```

Comment:
maybe `PROJ_ROOT = Path(__file__).resolve().parent.parent.parent` to stay consistent?

Reply:
yes that looks better, but dumb question.. could you clarify a bit on staying consistent to what?

Reply:
ah you mean with itself!

Reply:
yes exactly hehe. Sorry should have been more clear here.